deeposlandia/keras_feature_detection.py

```diff
@@ -0,0 +1,96 @@
+
+"""Design a feature detection model with Keras API
+"""
+
+import h5py
+
+import keras as K
+from keras.applications import VGG16
+from deeposlandia.network import ConvolutionalNeuralNetwork
+
+class FeatureDetectionNetwork(ConvolutionalNeuralNetwork):
+    """Class that encapsulates feature detection network creation
+
+    Inherits from `ConvolutionalNeuralNetwork`
+
+    Attributes
+    ----------
+    _network_name : str
+        Name of the network
+    _image_size : integer
+        Input image size (height and width are equal)
+    _nb_channels : integer
+        Number of input image channels (1 for greyscaled images, 3 for RGB images)
+    _nb_labels : integer
+        Number of classes in the dataset glossary
+    X : tensor
+        (batch_size, image_size, image_size, nb_channels)-shaped input tensor; input image data
+    Y : tensor
+        (None, nb_classes)-shaped output tensor, built as the output of the last network layer
+    """
+
+    def __init__(self, network_name="mapillary", image_size=512, nb_channels=3,
+                 nb_labels=65, learning_rate=1e-4, architecture="simple"):
+        ConvolutionalNeuralNetwork.__init__(self, network_name, image_size,
```

Comment:
I think you should use the `super()` function for the inheritance.

Reply:
Changed to `super().__init__(network_name, image_size, ...)`.
This format is only `Python-3.x` compliant, however I guess the code is still Python3-oriented. We mention Python3 in the `setup.py`, I do not if we have to specify again elsewhere.